projects/canopy/src/lib/forms/date/date-field.notes.ts

```diff
@@ -17,37 +17,43 @@ Import the Date Input Module into your application:
 and in your HTML:
 
 ~~~html
-<lg-date-field formControlName="date">
-  Date of birth
-  <lg-hint>For example, 12 06 1983</lg-hint>
-  <lg-validation *ngIf="isControlInvalid(date, validationForm)">
-  <ng-container *ngIf="date.hasError('invalidField')">
-    Enter a valid {{ date.errors.invalidField }}
-  </ng-container>
-  <ng-container *ngIf="date.hasError('invalidFields')">
-    Enter a valid {{ date.errors.invalidFields[0] }} and
-    {{ date.errors.invalidFields[1] }}
-  </ng-container>
-  <ng-container *ngIf="date.hasError('requiredField')">
-    Date of birth must include a {{ date.errors.requiredField }}
-  </ng-container>
-  <ng-container *ngIf="date.hasError('requiredFields')">
-    Date of birth must include a {{ date.errors.requiredFields[0] }} and
-    {{ date.errors.requiredFields[1] }}
-  </ng-container>
-  <ng-container *ngIf="date.hasError('invalidDate')">
-    Enter a valid date of birth
-  </ng-container>
-  <ng-container *ngIf="date.hasError('futureDate')">
-    Date must be in the past
-  </ng-container>
-  <ng-container *ngIf="date.hasError('required')">
-    Enter a date of birth
-  </ng-container>
-</lg-validation>
-</lg-date-field>
+<form [formGroup]="form" (ngSubmit)="..." #validationForm="ngForm">
+  <lg-date-field formControlName="date">
+    Date of birth
+    <lg-hint>For example, 12 06 1983</lg-hint>
+    <lg-validation *ngIf="isControlInvalid(date, validationForm)">
+      <ng-container *ngIf="date.hasError('invalidField')">
+        Enter a valid {{ date.errors.invalidField }}
+      </ng-container>
+      <ng-container *ngIf="date.hasError('invalidFields')">
+        Enter a valid {{ date.errors.invalidFields[0] }} and
+        {{ date.errors.invalidFields[1] }}
+      </ng-container>
+      <ng-container *ngIf="date.hasError('requiredField')">
+        Date of birth must include a {{ date.errors.requiredField }}
+      </ng-container>
+      <ng-container *ngIf="date.hasError('requiredFields')">
+        Date of birth must include a {{ date.errors.requiredFields[0] }} and
+        {{ date.errors.requiredFields[1] }}
+      </ng-container>
+      <ng-container *ngIf="date.hasError('invalidDate')">
+        Enter a valid date of birth
+      </ng-container>
+      <ng-container *ngIf="date.hasError('futureDate')">
+        Date must be in the past
+      </ng-container>
+      <ng-container *ngIf="date.hasError('required')">
+        Enter a date of birth
+      </ng-container>
+    </lg-validation>
+  </lg-date-field>
+
+  <button type="submit">Submit</button>
+</form>
 ~~~
 
+**Note: for the validation to work properly it's very important to include the \`ngForm\` on the form tag and for the submit button to be within the form.**
```

Comment:
I know we now have two places in the same notes where we repeat this sentence but i'm keen to leave both of them as it's such an important step.

Reply:
I agree 👍